export_img.py

```diff
@@ -37,33 +34,21 @@ def find_inkscape_path():
 
 inkscape = find_inkscape_path()
 
-def create_png_directory():
-    """
-    Check if png directory exist, if not create new one
-    """
-    dir = "../png"
-    if os.path.isdir(dir):
-        pass
-    else:
-        os.mkdir(dir)
-
 def export_svg_png():
     """
     Export img.svg to img.png with Inkscape
     """
-    for img in base.glob("*.svg"):
+    for img in Path("./img").glob("**/*.svg"):
         name = str(img)
         parts = []
-        new_name = ""
-
         for part in img.with_suffix(".png").parts:
             if part == "svg":
                 parts.append("png")
             else:
                 parts.append(part)
-            new_name = str(Path(*parts))
-        subprocess.run([inkscape, name, "--export-png=" + "../png/" + new_name, "--export-area-page"], check = True,)
+        new_name = Path(*parts)
```

Comment:
Hvezdicka je v kodu novinka a neni soucasti zacatecnickeho kurzu. Videla bych zde komentar s odkazem bud na nas blog, nebo jinou stranku, kde se hvezdicka srozumitelne vysvetluje.

Reply:
A co že to není součástí začátečnického kurzu? S tím nemáme nic společné. Kdo chce tak si to přečte na blogu že jsme to probírali. To bysme museli potom vysvětlovat všechno v kódu.

Reply:
@ivet1987, podle mě to není problém, že se to vysvětlí v kódu: zatím ty nové věci, jako dictionary comprehension, nebo `__new()__` se okomentovaly dodatečně s odkazem. Pokud někdo bude chtít číst tento skript (třeba nějaká PyLady po začátečnickém kurzu, kterou zajímá, co může napsat se svými znalostmi, nebo, co je víc pravděpodobně: někdo z nás za dva měsíce), bude si muset opět googlit, co je to zde za věc a jak se to používá. Je lepší mít kvalitní odkaz hned po ruce.

Reply:
Můžeš kdyžtak odkázat na "Proměnný počet argumentů" v Lumírově lekci o argumentech funkcí: https://github.com/frenzymadness/Advanced_PyLadies/blob/master/03_functions_arguments/functions_arguments.ipynb
Nebo na hutnou oficiální dokumentaci: https://docs.python.org/3/reference/expressions.html#calls